pages/coronavirus-research.md

```diff
@@ -0,0 +1,286 @@
+- Receive more regular health check-ins as part of your study visits
+- Get early access to a new vaccine or treatment
+
+No one can predict the exact outcome of a research study. You may or may not see a direct benefit to your health. But your participation does matter.
+
+### More about participating in VA research
+
+<ul class="usa-accordion">
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="participate">What should I expect if I'm eligible to participate in a study?</button>
+<div id="participate" class="usa-accordion-content">
+
+The first thing you should know is that signing up for our volunteer list doesn’t mean you’re committing to join a research study. It just means you want to learn more. Here’s what to expect if you decide to sign up, and if we think you may be eligible for a specific study.
+
+#### Before the study
+
+If we contact you about a study, we’ll give you written information about the study. We’ll also talk with you about all of the details of the study, including possible risks, to help you decide whether you want to join. This process is called informed consent.
+
+**At this time, you'll want to ask questions like:**
+
+- Who's sponsoring this study?
+- Who reviewed or approved this study? 
+- What could happen to my health, good or bad, if I take part in this study?
+
+Get more questions to ask in our [Volunteering in Research (PDF)](https://www.research.va.gov/resources/pubs/docs/volunteering-in-research-brochure.pdf) brochure.
+
+If you decide you want to join the study, our researchers will determine if you’re eligible to participate.
+
+**As part of this, you may:**
+
+- Answer questions about your medical history
+- Get a physical exam
+- Get a blood test
+
+#### During the study
+
+If you’re eligible, you can then join the study. Remember, it’s always your choice and you can decide to leave a study at any time.
+
+**During the study, we may ask you to:**
+
+- **Go to a VA health facility near you.** For most studies, you’ll need to go for 10 or more visits over 1 to 2 years.
+- **Get injections or infusions.** You may get the study product we’re researching, or you may get a placebo. A placebo is a liquid with no active ingredients.
+- **Keep track of how you feel.** We’ll likely ask you to write down how you feel for about a week after you get each injection or infusion. The study staff will also contact you to ask how you’re feeling.
+
+**Note:** We won’t expose you to the virus that causes COVID-19 as part of any studies. If you do get sick during the study, we’ll work with you to make sure you get the care you need.
+
+#### After the study
+
+When the study is complete, we may ask you if we can contact you again for another study. You can say no if you’d prefer.
+
+We’ll also share the results of the study with you and your community. We won’t include your or any other participants’ name or any information that can be used to identify you.
+
+</div>
+</li>
```

Comment:
Not sure if these 2 lines (123, 124) are needed, might create unintended styling issues 🙂

Reply:
They're there to separate the accordions. But definitely let me know if they're not correct. Thanks!

Reply:
They look great! 🎉